index.js

```diff
@@ -124,6 +122,7 @@ module.exports = {
           fill: ['currentColor', 'transparent', 'inherit'],
           stroke: ['currentColor', 'transparent', 'inherit'],
         },
+        disableFix: true,
```

Comment:
For `stylelint-declaration-strict-value` to be able to auto-fix issues a fix function must be provided. Disabling autofix (consumers can create their own function if they want).

Reply:
Ah yes I had seen this one previously and wasn't sure what the fix was so thank you for finding it!